inst/qml/Descriptives.qml

```diff
@@ -192,6 +192,20 @@ Form
 					name: "sdCiLevel"
 					label: qsTr("Width")
 				}
+
+				DropDown
+				{
+					name: "sdCiMethod"
+					label: qsTr("Method")
+					id: sdCiMethod
+					indexDefaultValue: 0
+					info: qsTr("How should the confidence interval be computed? By default, we use a chi-sqaured model. The alternative option is `Bootstrap`")
```

Comment:
```suggestion
					info: qsTr("How should the confidence interval be computed? By default, we use a chi-squared model. The alternative option is `Bootstrap`")
```
same below for variances.

I wonder if "Chi-squared model" is the right name. Perhaps we should just call this the "analytical" solution or "normal" approach and then leave the details for the help file?